webapp/cube/views.py

```diff
@@ -153,7 +153,7 @@ def get_microcerts():
     # also this avoids loading contracts api where not required
     is_test_backend = (
         current_app.config["CONTRACTS_TEST_API_URL"]
-        if flask.request.args.get("test_backend", "false")
+        if flask.request.args.get("test_backend", "true")
```

Comment:
Why did you switch this around?

Reply:
Because I assume that production is never going to have `test_backend=true`? so not setting it means it wiill be false?

Reply:
Reversed. Just checked and it seems to work correctly.